.github/workflows/preview.yml

```
@@ -35,8 +36,25 @@
         env:
           NEXT_PUBLIC_API_URL: ${{ secrets.NEXT_PUBLIC_API_URL }}
 
+      - name: Check AuthGuard content
+        run: |
+          echo "AuthGuard content:"
+          cat apps/web/app/components/auth/AuthGuard.tsx
+
       - name: Build
-        run: pnpm -filter=web build
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)